test/test_browser.py

```diff
@@ -4256,21 +4256,26 @@ def test_pthread_custom_pthread_main_url(self):
     delete_file('test.worker.js')
     self.run_browser('test2.html', '/report_result?exit:0')
 
-  # Test that if the main thread is performing a futex wait while a pthread needs it to do a proxied operation (before that pthread would wake up the main thread), that it's not a deadlock.
+  # Test that if the main thread is performing a futex wait while a pthread
+  # needs it to do a proxied operation (before that pthread would wake up the
+  # main thread), that it's not a deadlock.
   @requires_threads
   def test_pthread_proxying_in_futex_wait(self):
     self.btest_exit('pthread/test_pthread_proxying_in_futex_wait.cpp', args=['-O3', '-pthread', '-sPTHREAD_POOL_SIZE'])
 
   # Test that sbrk() operates properly in multithreaded conditions
   @requires_threads
-  def test_pthread_sbrk(self):
-    for aborting_malloc in [0, 1]:
-      print('aborting malloc=' + str(aborting_malloc))
-      # With aborting malloc = 1, test allocating memory in threads
-      # With aborting malloc = 0, allocate so much memory in threads that some of the allocations fail.
-      self.btest_exit('pthread/test_pthread_sbrk.cpp', args=['-O3', '-pthread', '-sPTHREAD_POOL_SIZE=8', '-sABORTING_MALLOC=' + str(aborting_malloc), '-DABORTING_MALLOC=' + str(aborting_malloc), '-sINITIAL_MEMORY=128MB'])
+  @parameterized({
+    '': (['-DABORTING_MALLOC=0', '-sABORTING_MALLOC=0'],),
```

Comment:
```suggestion
    '': (['-DABORTING_MALLOC=0'],),
```

Reply:
I think this flag is needed to actually disable the ABORTING_MALLOC setting.

Reply:
Oh, sorry, my brain didn't realize one was `-s` and one `-D`...